db/migrate/20170720194715_create_middleware_jdr_reports.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMiddlewareJdrReports < ActiveRecord::Migration[5.0]
+  def change
+    create_table :middleware_jdr_reports do |t|
```

Comment:
jdr sounds very java specific.  If there were an alternate non-java middleware provider what would the equivalent/generic term be?